lib/utils/fixEmptyLineBefore.mjs

```diff
@@ -0,0 +1,39 @@
+import { assert } from './validateTypes.mjs';
+
+/** @typedef {import('stylelint').Range} Range */
+/** @typedef {import('postcss').Node} Node */
+/** @typedef {(node: Node, newline: string) => Node} Callback */
+
+/**
+ * fix callback for *-empty-line-before rules
+ * @param {object} o
+ * @param {Node} o.node
+ * @param {string=} o.newline
+ * @param {Callback} o.mutate
+ * @param {Range} [o.range]
+ * @returns {Range}
+ */
+export default function fixEmptyLineBefore({ node, newline, mutate, range }) {
```

Comment:
[suggest] It seems more accurate to use plural (`line` or `lines`). Any thoughts?

```suggestion
export default function fixEmptyLinesBefore({ node, newline, mutate, range }) {
```